src/server/cmd/pachd/main.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os"
 	"strings"
 
+	units "github.com/docker/go-units"
```

Comment:
I'm assuming this was already vendored in?  Just making sure we weren't adding it for this purpose alone.